src/ice_shelf/MOM_ice_shelf_dynamics.F90

```diff
@@ -579,9 +579,9 @@ subroutine initialize_ice_shelf_dyn(param_file, Time, ISS, CS, G, US, diag, new_
     CS%id_col_thick = register_diag_field('ice_shelf_model','col_thick',CS%diag%axesT1, Time, &
        'ocean column thickness passed to ice model', 'm', conversion=US%Z_to_m)
     CS%id_visc_shelf = register_diag_field('ice_shelf_model','ice_visc',CS%diag%axesT1, Time, &
-       'viscosity', 'm', conversion=1e-6*US%Z_to_m)
+       'vi-viscosity', 'Pa s-1 m', conversion=US%Z_to_m) !vertically integrated viscosity
```

Comment:
The unit conversion factor here seems to be at odds with the new units, but it could be correct if this diagnostic is only being partially rescaled by this conversion factor.  Please confirm that this diagnostic does not change when you use a variety of values of Z_RESCALE_POWER, T_RESCALE_POWER, L_RESCALE_POWER, and R_RESCALE_POWER.

Reply:
The conversion has been corrected. I 'believe' it's correct, the SI units of this diagnostic are Pa s-1 m.